[O11y][MongoDB Atlas] Make GA

harnish-elastic opened the pull request:
- Enhancement

<!-- Type of change
Please label this PR with one of the following labels, depending on the scope of your change:
- Bug
- Enhancement
- Breaking change
- Deprecation
-->

## Proposed commit message

- Make MongoDB Atlas GA

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [x] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Related issues

<!-- Recommended
Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.

- Closes #123
- Relates #123
- Requires #123
- Supersedes #123
-->
- Relates https://github.com/elastic/obs-integration-team/issues/201

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/24494)
* Commit: af25307a8fbc8fd7a6cdaf910e0f5720b63c6735

### Failed CI Steps
* [:pipeline::arrow_up: Upload Pipeline: .buildkite/pipeline.yml](https://buildkite.com/elastic/integrations/builds/24494#019614e3-b2e9-473f-86ef-3921823c7022)

cc @harnish-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/24494","number":24494,"commit":"af25307a8fbc8fd7a6cdaf910e0f5720b63c6735"}],"number":24494}
buildkite-pr-comment-->

ishleenk17 commented on packages/mongodb_atlas/changelog.yml:
```diff
@@ -1,4 +1,9 @@
+- version: "1.0.0"
+  changes:
+    - description: Make GA.
+      type: bugfix
```
```suggestion
      type: enhancement
```

ishleenk17 commented on packages/mongodb_atlas/changelog.yml:
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.0.0"
+  changes:
+    - description: Make GA.
```
```suggestion
    - description: Make MongoDB Atlas GA.
```

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/24509)
* Commit: 10050577945cb562e47ac568b3fca85b7dff0310

### History
* :broken_heart: [Build #24494](https://buildkite.com/elastic/integrations/builds/24494) failed af25307a8fbc8fd7a6cdaf910e0f5720b63c6735

cc @harnish-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/24509","number":24509,"commit":"10050577945cb562e47ac568b3fca85b7dff0310"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/24494","number":24494,"commit":"af25307a8fbc8fd7a6cdaf910e0f5720b63c6735"}],"number":24509}
buildkite-pr-comment-->

harnish-elastic commented:
@daniela-elastic I have addressed review comments for the README update [PR](https://github.com/elastic/integrations/pull/13420#discussion_r2033131308). Thank you!

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/24592)
* Commit: 0de3b839474a93c105d10712f4374706017c7287

### History
* :green_heart: [Build #24509](https://buildkite.com/elastic/integrations/builds/24509) succeeded 10050577945cb562e47ac568b3fca85b7dff0310
* :broken_heart: [Build #24494](https://buildkite.com/elastic/integrations/builds/24494) failed af25307a8fbc8fd7a6cdaf910e0f5720b63c6735

cc @harnish-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/24592","number":24592,"commit":"0de3b839474a93c105d10712f4374706017c7287"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/24509","number":24509,"commit":"10050577945cb562e47ac568b3fca85b7dff0310"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/24494","number":24494,"commit":"af25307a8fbc8fd7a6cdaf910e0f5720b63c6735"}],"number":24592}
buildkite-pr-comment-->

ishleenk17 reviewed (approved):
Looks good!
